client/src/tests/elements/Pie.test.tsx

```diff
@@ -40,11 +39,13 @@ describe('Pie', () => {
           subtitle);
         pie = TestUtils.renderIntoDocument(pieElement);
         TestUtils.isElementOfType(pie, 'div');
+
+        // Adding a timeout to make sure Flux cycle is complete
+        setTimeout(done, 100);
+        
+      } catch (e) {
+        return done(e);
```

Comment:
what about failing the test here and also within the settimout (if i got it right, those are the negative flows)